.vscode.dist/launch.json

```diff
@@ -0,0 +1,24 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Run",
```

Comment:
Since these are Windows-specific, maybe they should be called "Run (Windows)" and "Build (Windows)"?

Reply:
Makes sense. I tried to make it work on other OSes now, but didn't test it. I hope it at least provides a good starting point for developers on these platforms.